sdk/identity/Azure.Identity/tests/samples/CustomCredentialSnippets.cs

```diff
@@ -68,7 +68,7 @@ public void ConfidentialClientCredentialUsage()
             #endregion
         }
 
-        #region Snippet:OnBehalfOfCredential
+        //#region Snippet:OnBehalfOfCredential
```

Comment:
Is this intended to remain commented for this release?

Reply:
Yes, I left the snippet in code files even though it's been removed from the docs in case just in case we need to support customers who had used this before we added the OnBehalfOfCredential to the library as it's a bit different from the sample we used to have. But, if you feel strongly about it we can remove it.